build.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+##############################################################################
+# Setup script for CEPCSW:
+# - build the cepcsw
+#
+# Usage:
+# $ bash setup.sh
```

Comment:
To make things simple, I would suggest to chmod u+x for build.sh, and only use this command to build: 

$ ./build.sh  (with options in the future)

Reply:
Fixed.